tasks/models.py

```diff
@@ -18,19 +17,22 @@ class Task(models.Model):
     end_time = models.DateTimeField(null=True)
     deleted_at = models.DateTimeField(null=True)
     level = models.CharField(max_length=1, default="")
+    situations_m2m = models.ManyToManyField("Situation", through="TakeTask")
+    member_id = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE, default=1)
 
     objects = TaskManager()
 
     def delete(self):
         self.deleted_at = timezone.now()
         self.save()
 
-
-class TakeTask(models.Model):
-    pass
-
-
 class Situation(models.Model):
```

Comment:
```suggestion
class Situation(models.Model):
```
這個表主要是做什麼的？是任務的狀態嗎？因為那個 `col_name` 的命名看不太出來是什麼